framework/src/material_image_helpers.cpp

```diff
@@ -2,247 +2,371 @@
 
 namespace gvk
 {
-
-	std::tuple<std::vector<material_gpu_data>, std::vector<avk::image_sampler>> convert_for_gpu_usage(
-		const std::vector<gvk::material_config>& aMaterialConfigs, 
+	static inline std::tuple<std::vector<material_gpu_data>, std::vector<avk::image_sampler>> convert_for_gpu_usage_cached(
```

Comment:
This should not be `static inline` - especially not in a `.cpp` file?! This method is (probably/hopefully?!) declared `extern` in the header file. If so, its definition here in the `.cpp` file should not have any of such declarations (i.e. neither `static`, nor `inline`, nor `extern` in this case here in the `.cpp`).